example/k3d/smoke/main.jsonnet

```diff
@@ -23,6 +25,11 @@ local new_crow(name, selector) =
     },
   });
 
+local chaosFrequency = std.extVar("chaos_frequency");
+local mutationFrequency = std.extVar("mutation_frequency");
```

Comment:
Can we give these a default if they're not defined? I do local applies a lot when playing around with the smoke environment.

Reply:
@rfratto From what I can tell, `std.extVar` will return an error if this value is not passed and I'm not sure how to handle that error. Is there a pattern I can use here for assigning default values to external vars?

Reply:
If we're ok modifying these values in the main.jsonnet, I could remove them from the smoke script and not pass them in as external vars. Let me know what you think.